.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Neat! Wasn't aware of this one

Reply:
We did that in SLB as well to avoid useless package-lock files :P They have their uses, but what brainfart that made NPM turn it on by default is beyond me.  Shrinkwrap > package-lock :P